src/integrationTest/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/ProfessionalUserServiceIntegrationTest.java

```diff
@@ -83,8 +103,36 @@ void shouldRefreshUsers() {
         assertEquals("ACTIVE", refreshedUser.getOrganisationStatus());
         assertArrayEquals(new String[]{"SOLICITOR_PROFILE", "2"},
                 refreshedUser.getOrganisationProfileIds());
-        assertFalse(refreshedUser.getActive());
+        //assertFalse(refreshedUser.getActive()); why did Sanjay think this should be false, it is true?
```

Comment:
why did Sanjay think this should be false, it is true?

Reply:
When we've processed records successfully they should be marked as `active` `false`.